SwiftFlow/CoreTypes/Reducer.swift

```diff
@@ -25,3 +25,23 @@ extension Reducer {
     }
 
 }
+
+protocol PartialReducer: Reducer {
+    
+    typealias PartialStateType: StateType
+    typealias HandledActionType: Action
+    
+    func get(state: StateType) -> PartialStateType
+    func set(partialState: PartialStateType, originalState: StateType) -> StateType
+    func handleSupportedAction(partialState: PartialStateType, supportedAction: HandledActionType) -> PartialStateType
```

Comment:
Line Length Violation: Line should be 100 characters or less: currently 118 characters (line_length)